CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+5.1.3 - 2025-08-22
```

Comment:
We use the release date in the changelog and our next scheduled release is next week.
```suggestion
5.1.3 - 2025-09-01
```

Also, please update the version number in the plugin.json file to match your new changelog entry.